install_config/router/f5_router.adoc

```diff
@@ -127,6 +129,14 @@ $ oadm router \
 ====
 endif::[]
 
+As with the HAProxy router, the `oadm router` command creates the service and
+deployment configuration objects, and thus the replication controllers and
+pod(s) in which the F5 router itself runs. The replication controller restarts
+the F5 router in case of crashes. Because the F5 router is only watching routes
+and endpoints and configuring *F5 BIG-IP®* accordingly, running the F5 router in
```

Comment:
Even though this specific case doesn't need it - the F5 router does watch nodes as well. Maybe just say "Because the F5 router is watching routes, endpoints and nodes ... ".